requirements/requirements-testing.txt

```diff
@@ -7,7 +7,8 @@ freezegun>=0.3.7,<0.4
 mock>=2.0,<3.0
 
 # PyTest for running the tests.
-pytest
+#: pinned until pytest-asyncio will be compatible with pytest==`4.1.0`
```

Comment:
looks like it [already is](https://github.com/pytest-dev/pytest-asyncio#0100-2019-01-08)

Reply:
Will clean it up!